networkx/algorithms/tests/test_cycles.py

```diff
@@ -212,3 +214,44 @@ def test_dag(self):
                       find_cycle, G, orientation='original')
         x = list(find_cycle(G, orientation='ignore'))
         assert_equal(x, [(0,1,FORWARD), (1,2,FORWARD), (0,2,REVERSE)])
+
+
+class TestMinimumCycles:
```

Comment:
Inherit from `object` for Python 2 compatibility purposes: `class TestMinimumCycles(object):`